lib/yoti/doc_scan/session/create/notification_config.rb

```diff
@@ -0,0 +1,142 @@
+# frozen_string_literal: true
+
+module Yoti
+  module DocScan
+    module Session
+      module Create
+        #
+        # Configures call-back Notifications to some backend endpoint provided
+        # by the Relying Business.
+        #
+        # Notifications can be configured to notified a client backend of certain
+        # events, avoiding the neeed to poll for the state of the Session.
```

Comment:
Looks like `ruby` isn't checked by default in the "Code Spell Checker" extension settings! Think this is the only one I found (in your new code, there are a few others in old code)
```suggestion
        # events, avoiding the need to poll for the state of the Session.
```

Reply:
Fixed in b7a7c43